schema/askbot.csv

```diff
@@ -1,4 +1,5 @@
-name,type
+name,type
+accepted_answer_id,long

```

Comment:
I would add another field: `is_accepted_answer`, long
From the point of view of answers it will allow to identify the accepted ones.